plugin/xtrabackup/plugin.go

```diff
@@ -302,15 +320,18 @@ func (p XtraBackupPlugin) Backup(endpoint plugin.ShieldEndpoint) error {
 
 	plugin.DEBUG("Executing: `%s`", cmdString)
 	if err = plugin.ExecWithOptions(opts); err != nil {
+		fmt.Fprintf(os.Stderr,"@R{\u2717 Create backup files}\n")
 		return err
 	}
+	fmt.Fprintf(os.Stderr,"@G{\u2713 Create backup files}\n")
 
 	// create and return archive
 	cmdString = fmt.Sprintf("%s -cf - -C %s .", xtrabackup.Tar, targetDir)
 	if err = plugin.Exec(cmdString, plugin.STDOUT); err != nil {
+		fmt.Fprintf(os.Stderr,"@R{\u2717 Create archive}\n")
 		return err
 	}
-
+	fmt.Fprintf(os.Stderr,"@G{\u2713 Create archive}\n")
```

Comment:
These log messages read better as "Creating archive failed: <err>" and "Created archive"